Base/QTGUI/qSlicerExtensionsManagerWidget.cxx

```diff
@@ -453,7 +453,7 @@ void qSlicerExtensionsManagerWidget::onModelUpdated()
     }
 
   d->tabWidget->setTabText(manageExtensionsTabIndex,
-                           QString("Manage Extensions (%1)").arg(managedExtensionsCount));
+                           tr("Manage Extensions (%1)").arg(managedExtensionsCount));
```

Comment:
Thanks for your feedback, it's added now.

However, I notice that lupdate doesn't handle developper comments the same way. In fact, one can use :

- [translator comments](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) like `//: developer comments` or `/*: developer comments */`
-  the old fashion [ _**tr**_ disambiguation argument](https://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation) like `tr("translatable string", "developer comments")`

Using translator comments will result in a `<extracomment>` tag in the translation file while the use of the disambiguation parameter will appear as a `<comment>` tag.

As discussed in the last meeting, I will test them both on Weblate to verify how they are handled so that to see which comment style we may consider in the project.

Reply:
> the old fashion [tr disambiguation argument](https://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation) like tr("translatable string", "developer comments")

Disambiguation and comments are for entirely purposes. Disambiguation argument is used at runtime for getting different translation for the same source text. Comments are ignored at runtime.

Here we need to use comments, as we don't need to distinguish between several "Manage Extensions (%1)" strings.

It would be useful to see how comment and extracomment show up in weblate. That will help us decide which one to use in what cases.

Reply:
Got it, you're right.

I will test them both and Weblate and see how they are handled.

Thanks !

Reply:
After some tests, I realise that Weblate doesn't handle the `<comment>` and `<extracomment>` tags the same way.

- Disambiguation strings present on `<comment>` tags are just added to the context name of the string so that to constitute its `Key`. That key is displayed in the `String information` and the `Translation` sections of Weblate. Examples of such cases can be found [here](https://hosted.weblate.org/translate/3d-slicer/3d-slicer/fr/?offset=1&q=qSlicerWelcomeModuleWidget+extension+update+is+available&sort_by=context&checksum=) or [there](https://hosted.weblate.org/translate/3d-slicer/3d-slicer/fr/?checksum=e2279a7500de6c73&sort_by=-priority,position).
- Translator comments present on `<extracomment>` tags are displayed as `String source description` in the `String information` section of Weblate. Examples of such cases can also be found [here](https://hosted.weblate.org/translate/3d-slicer/3d-slicer/fr/?checksum=9e57190062ff79e2&sort_by=context) or [there](https://hosted.weblate.org/translate/3d-slicer/3d-slicer/fr/?offset=1&q=qSlicerSequencesModuleWidget+Limited&sort_by=context&checksum=).

I think then we should use translator comments to integrate comments for translators.